lib/connection.js

```diff
@@ -193,6 +197,10 @@ class Connection extends EventEmitter {
   _heartbeatCallback(interval) {
     this.transport.send('{}');
     this.setTimeout(interval, this._heartbeatCallbackInstance);
+
+    if (process.env.NODE_ENV !== 'production') {
+      this.emit('sentPacket', this._heartbeatMessage);
```

Comment:
@lundibundi, I think, @nechaido meant that there should also be `heartbeat` event, which will be always emitted in both development and production modes.
However, I don't see think we need this, emitting this messages in development mode only seems enough.